actions/loadconfig.py

```diff
@@ -1,29 +1,30 @@
 from lib.action import NapalmBaseAction
 
+from napalm.base.exceptions import MergeConfigException
+from napalm.base.exceptions import ReplaceConfigException
+
 
 class NapalmLoadConfig(NapalmBaseAction):
     """Load configuration into network device via NAPALM
     """
 
-    def run(self, config_file, method, inline_transfer, **std_kwargs):
-
-        if not method:
-            method = 'merge'
-        else:
-            method = method.lower()
-            if method not in ["merge", "replace"]:
-                raise ValueError(('{} is not a valid load method, use: '
-                                  'merge or replace').format(method))
-
-        with self.get_driver(**std_kwargs) as device:
-            # inline_transfer: If set it becomes True, else False
-            device.inline_transfer = inline_transfer
-
-            if method == "replace":
-                device.load_replace_candidate(filename=config_file)
-            else:
-                device.load_merge_candidate(filename=config_file)
-
-            device.commit_config()
+    def run(self, config_file, config_text, method, inline_transfer, **std_kwargs):
+        try:
```

Comment:
I think it's better to ensure here either of `config_file` or `config_text` is given and return a helpful text if it's not.

```
if config_file is None and config_text is None:
    raise ValueError('Specify either config_file or config_text')
```

Reply:
Thank you for your review, I agree with you.